### PR TITLE
RequestId: use UUID1 as trace and span ID

### PR DESCRIPTION
**Pull request by macbre**
```
composer.phar require ramsey/uuid
```

``` php
> echo Ramsey\Uuid\Uuid::uuid1();
c804f360-f1a6-11e5-a603-00163e046284
```

``` json
{
      "app_name": "mediawiki",
      "app_version": "dev",
      "wiki_dbname": "plpoznan",
      "wiki_id": "5915",
      "http_url_path": "/wikia.php",
      "http_method": "GET",
      "http_url_domain": "poznan.macbre.wikia-dev.com",
      "http_url": "http://poznan.macbre.wikia-dev.com/wikia.php",
      "client_ip": "10.14.30.28",
      "span_id": "d09dd88e-f1a6-11e5-8db2-00163e046284",
      "trace_id": "c5265c6a-f1a6-11e5-9417-00163e046284"
}
```

@wladekb / @pchojnacki

### CODE REVIEW

File: lib/Wikia/src/Tracer/RequestId.php [hunk 1]
```diff
@@ -68,24 +70,21 @@ function getRequestId() {
...
 	/**
-	 * Return 23 characters long unique ID + mw prefix
-	 * e.g. mw5405bb3d129e76.46189257
+	 * Return timestamp-based UUID (e.g. 8454441a-f0e1-11e5-9c4a-00163e046284)
 	 *
 	 * @return string
 	 */
 	public static function generateId() {
-		return uniqid( 'mw', true );
+		return Uuid::uuid1()->toString();
```
Comment (pawelchcki) [inline lib/Wikia/src/Tracer/RequestId.php:78]: :+1:  for using uuid1 as they include timestamp that can be pased out

**Comment by pawelchcki** [comment]
:+1:

**Comment by pawelchcki** [comment]
Although it looks like test from this PR is failing:

``````
1) RequestIdTest::testIsValidId with data set #2 ('mw5405bb3c76f364.47661604', true)
Failed asserting that false matches expected true.

/var/lib/jenkins/workspace/unitTests_php_branches/app/lib/Wikia/tests/Tracer/RequestIdTest.php:26```
``````

**Comment by pawelchcki** [comment]
Otherwise LGTM

**Comment by macbre** [comment]
Tests fixed in b293b39 and improved in 2bc675d

**Comment by pawelchcki** [comment]
:shipit:

**Pull request merged**